.github/drafts/15938/15938_Audit_Logs_Draft.md

```diff
@@ -0,0 +1,127 @@
+        "email": "john@appsmith.com",
+        "name": "John Doe"
+    },
+    "resource": {
+        "id": "62bc0f11545c4c00b19d5c59",
+        "type": "Datasource",
+        "name": "Movies"
+    },
+    "app": {
+        "id": "62bc0f11545c4c00b19d5c68",
+        "name": "Standup App",
+        "git": {
+            "branch": "feat/new-ui",
+            "default": "master"
+        }
+    },
+    "workspace": {
+        "id": "62bc0f11545c4c00b19d5c64",
+        "name": "Appsmith Internal Apps"
+    },
+    "metadata": {
+        "appsmithVersion": "1.7.5",
+        "createdAt": "2022-06-29T08:36:33.507+00:00"
+    }
+}
+```
+
+### Tracked events
+
+The following table shows the events that appear in Appsmith's audit logs. In any given log record, you can find the event's name on the record's `event` property.
+
+| Event Name | Description |
+|------------|-------------|
+| **workspace.created** | Logged when a new workspace is created. |
+| **workspace.updated** | Logged when the details of an existing workspace are changed. |
+| **workspace.deleted** | Logged when a workspace is deleted. |
+| **application.created** | Logged when a new application is created. |
+| **application.updated** | Logged when a user edits an existing application. |
+| **application.deleted** | Logged when an application is deleted. |
+| **application.imported** | Logged when a new application is created via importing with Git or JSON. |
+| **application.exported** | Logged when an existing application is exported via Git or JSON. |
+| **application.cloned** | Logged when a new application is created via cloning. |
+| **application.forked** | Logged when an existing application is forked. |
+| **application.deployed** | Logged when a new version of an application is deployed/published. |
+| **page.created** | Logged when a new page is created. |
+| **page.viewed** | Logged when a user, including any anonymous public users, views a page. |
+| **page.updated** | Logged when a user edits a page. |
+| **page.deleted** | Logged when a page is deleted. |
+| **datasource.created** | Logged when a new datasource is created. |
+| **datasource.updated** | Logged when a user edits an existing datasource's configuration. |
+| **datasource.deleted** | Logged when a datasource is deleted. |
+| **query.created** | Logged when a new query is created. |
+| **query.updated** | Logged when a user edits an existing query's configuration. |
+| **query.deleted** | Logged when a query is deleted. |
+| **query.executed** | Logged when an existing query runs. |
+| **user.logged_in** | Logged when a user logs into the platform. |
+| **user.logged_out** | Logged when a user logs out of the platform. |
+| **user.invited** | Logged when someone sends an invite to the platform. |
+| **user.signed_up** | Logged when a new user logs in for the first time. |
+| **instance_setting.updated** | Logged when a change is made to the platform instance's configuration. |
```

Comment:
Not sure if we use the platform in general so we could omit it.

@Pranay105 could probably weigh in here.